roles/logging-config/handlers/main.yml

```diff
@@ -2,3 +2,6 @@
 - name: restart logstash-forwarder
   service: name=logstash-forwarder state=restarted must_exist=false
   failed_when: false
+
+- name: restart filebeat
+  service: name=filebeat state=restarted must_exist=false
```

Comment:
currently must_exist doesn't function with systemd units, may need a failed_when: false in here.